configs/templates/unified-dev/spack.yaml

```diff
@@ -6,43 +6,47 @@ spack:
   include: []
 
   definitions:
-  - compilers: ['%aocc', '%apple-clang', '%gcc', '%intel']
+  - compilers: ['%aocc', '%apple-clang', '%gcc', '%intel', '%oneapi']
```

Comment:
Not in this PR, but at some point it might make sense to restructure our templates to not use the matrix approach since we're doing one compiler per environment.

Reply:
Absolutely. That's been my "next I'll work on this" task for a few months ...